packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -132,50 +111,31 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
               styles={props.commonButtonStyles}
             />
           )}
-          {
-            /* @conditional-compile-remove-from(stable): custom button injection */
-            <FilteredCustomButtons customButtonProps={customButtonProps} placement={'afterParticipantsButton'} />
-          }
+          {customButtons['afterParticipantsButton']}
           {options?.devicesButton !== false && (
             <Devices
               displayType={options?.displayType}
               increaseFlyoutItemSize={props.increaseFlyoutItemSize}
               styles={props.commonButtonStyles}
             />
           )}
-          {
-            /* @conditional-compile-remove-from(stable): custom button injection */
-            <FilteredCustomButtons customButtonProps={customButtonProps} placement={'afterOptionsButton'} />
-          }
+          {customButtons['afterOptionsButton']}
           {options?.endCallButton !== false && (
             <EndCall displayType={options?.displayType} styles={props.endCallButtonStyles} />
           )}
-          {
-            /* @conditional-compile-remove-from(stable): custom button injection */
-            <FilteredCustomButtons customButtonProps={customButtonProps} placement={'afterEndCallButton'} />
-          }
-          {
-            /* @conditional-compile-remove-from(stable): custom button injection */
-            <FilteredCustomButtons customButtonProps={customButtonProps} placement={'last'} />
-          }
+          {customButtons['afterEndCallButton']}
+          {customButtons['last']}
         </ControlBar>
       </Stack.Item>
     </Stack>
   );
 };
 
-/* @conditional-compile-remove-from(stable): custom button injection */
-const FilteredCustomButtons = (props: {
-  customButtonProps: CustomCallControlButtonProps[];
-  placement: CustomCallControlButtonPlacement;
-}): JSX.Element => {
-  return (
-    <>
-      {props.customButtonProps
-        .filter((buttonProps) => buttonProps.placement === props.placement)
-        .map((buttonProps, i) => (
-          <ControlBarButton {...buttonProps} key={`${buttonProps.placement}_${i}`} />
-        ))}
-    </>
-  );
+const onFetchCustomButtonPropsTrampoline = (
```

Comment:
`onFetchCustomButtonPropsTrampoline` 
What's the story behind this name?

Reply:
I mean I like a good trampoline but its also got me wondering

Reply:
Heh, old habit. `trampoline` is a term used to refer to helper functions for FFI bindings.

![image](https://user-images.githubusercontent.com/82062616/153513252-bf15506d-d47c-4df5-89ec-4160686df497.png)

from https://www.gnu.org/software/mit-scheme/documentation/stable/mit-scheme-ffi.pdf

Reply:
btw, 100% onboard with a more meaningful name for these kinds of functions for conditional compilation.